### PR TITLE
add targetEndDate vignette, add snake_case check

### PR DESCRIPTION
**Pull request by KimLopezGuell**

### CODE REVIEW

**Comment by KimLopezGuell** [comment]
#128 and #191

**Comment by KimLopezGuell** [comment]
@catalamarti the changes you told me about checkSnakeCase() are done. @edward-burn your empty table issue #200 should now also work, please check and report back

File: vignettes/targetEndDate.Rmd [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
```
Comment (catalamarti) [inline vignettes/targetEndDate.Rmd:1]: Can we implement this vignette in the current addCohortIntersect?

**Review by catalamarti — CHANGES_REQUESTED** [review]

File: R/addDemographics.R [hunk 2]
```diff
@@ -92,6 +92,12 @@ addDemographics <- function(x,
     cli::cli_abort("age, sex, priorHistory, futureObservation can not be FALSE")
   }
 
+  # check variable names
```
Comment (catalamarti) [inline R/addDemographics.R:95]: can you move this to checks?

File: R/addInObservation.R [hunk 3]
```diff
@@ -0,0 +1,71 @@
+#' Indicate if a certain record is within the observation period
```
Comment (catalamarti) [inline R/addInObservation.R:1]: this is inside addDemographics right?

**Comment by catalamarti** [comment]
Hey @KimLopezGuell I will accept the pull request because otherwise it will be annoying in the future for the name change that I am working on. Can you create a new PR later where you append the vignette to the existing one?

**Review by catalamarti — APPROVED** [review]

**Pull request merged**